python/tvm/topi/arm_cpu/tensor_intrin.py

```diff
@@ -403,8 +52,9 @@ def gemm_quantized(M, N, K, unroll, interleave, in_type, out_type):
         columns of the matrix B
     K: int
         columns of matrix A
+    unroll: boolean
```

Comment:
Couple of things, stylistically NumPy docstrings would leave a space either side of the colon and in Python the type would be 'bool'.
```suggestion
    unroll : bool
```